aws-fargate-cfn/op-scim-bridge.yaml

```diff
@@ -0,0 +1,392 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Deploys 1Password SCIM bridge using Amazon ECS and AWS Fargate. Provisions and
+  configures a VPC, 2 public subnets, an internet gateway, a route table, an
+  ALB, an ACM certificate, Route 53 DNS records, an AWS secret to store
+  credentials for your SCIM bridge, security groups, and required IAM roles.
+Metadata: 
+  AWS::CloudFormation::Interface: 
+    ParameterGroups:
+      - 
+        Label: 
+          default: ""
+        Parameters: 
+          - VPCCIDR
+          - Route53HostedZone
+          - DomainName
+          - SCIMBridgeVersion
+          - scimsession
+    ParameterLabels: 
+      VPCCIDR: 
+        default: "VPC CIDR"
+      Route53HostedZone:
+        default: "Route 53 hosted zone"
+      DomainName:
+        default: "1Password SCIM bridge domain name"
+      SCIMBridgeVersion:
+        default: "1Password SCIM bridge version"
+Parameters:
+  VPCCIDR:
+    Type: String
+    Default: 10.0.0.0/16
+    Description: A CIDR block for the VPC to be created
+  Route53HostedZone:
+    Type: 'AWS::Route53::HostedZone::Id'
+    Description: >-
+      The Route 53 hosted zone in which to create DNS records for ACM validation and the ALB
+  DomainName:
+    Type: String
+    Default: scim.example.com
+    Description: >-
+      A fully qualified domain name for your SCIM bridge; this must be in the
+      domain of the selected Route 53 hosted zone (where the record will be
+      created)
+  scimsession:
+    Type: String
+    Description: >-
+      The plain text contents of the scimsession file created during the
+      automated user provisioning setup in your 1Password account
+    NoEcho: true
+  SCIMBridgeVersion:
+    Type: String
+    Default: 'v2.8.1'
+    Description: >-
+      The tag of the 1Password SCIM bridge image to pull from Docker Hub (see
+      https://app-updates.agilebits.com/product_history/SCIM for detailed
+      release notes)
+Resources:
+  scimsessionSecret:
+    Type: 'AWS::SecretsManager::Secret'
+    Properties:
+      SecretString: !Base64
+        Ref: scimsession
+  ECSCluster:
+    Type: 'AWS::ECS::Cluster'
+    Properties:
+      CapacityProviders:
+        - FARGATE
+      DefaultCapacityProviderStrategy:
+        - CapacityProvider: FARGATE
+          Weight: 1
+  ECSTaskDefinition:
+    Type: 'AWS::ECS::TaskDefinition'
+    Properties:
+      RequiresCompatibilities:
+        - FARGATE
+      NetworkMode: awsvpc
+      Cpu: 512
+      Memory: 1024
+      ExecutionRoleArn: !GetAtt ExecutionRole.Arn
+      TaskRoleArn: !Ref TaskRole
+      ContainerDefinitions:
+        - Name: op-scim-bridge
+          Image: !Sub '1password/scim:${SCIMBridgeVersion}'
```

Comment:
Update line to include "v" from image tag, so the "v" does not need to included in the parameter as detailed in line 53.

Reply:
addressed in previous comment for line 53